tests/test_workflow.py

```diff
@@ -6,14 +6,36 @@
 MOCKED_OS_ENVIRON = "os.environ"
 
 
-def test_workflow_tumor_normal(
+def test_workflow_tumor_only_tga(
+    tumor_only_config, sentieon_install_dir, sentieon_license
+):
+    # GIVEN a sample config dict and snakefile
+    analysis_type = "single"
+    reference_genome = "hg19"
+    analysis_workflow = "balsamic"
+    snakefile = get_snakefile(analysis_type, analysis_workflow, reference_genome)
+    config_json = tumor_only_config
+
+    # WHEN invoking snakemake module with dryrun option
+    # THEN it should return true
```

Comment:
Since you're doing some refactoring, let's try to also start using some more meaningful comments
```suggestion
    # THEN the Snakemake workflow should have run successfully
```

Reply:
Fixed!